src/index.js

```diff
@@ -189,7 +189,7 @@ function listen(options, callback) {
     }
 
     if (opts.immediate) callback(mq);
-    mq.addEventListener('change', callback);
+    if (mq) mq.addEventListener('change', callback);
```

Comment:
There is no feedback that nothing will happen it will silently fail. There should be some hint that the listener will not be attached. Maybe a `console.warn` with a meaningful message.